components/common/confirmPurchaseModal/ConfirmPurchaseModal.vue

```diff
@@ -116,12 +114,10 @@ const totalRoyalties = computed(() =>
   ),
 )
 
-const totalWithRoyalties = computed(
-  () => totalNFTsPrice.value + totalRoyalties.value,
-)
+const { supportFee } = useBuySupportFee(urlPrefix, totalNFTsPrice)
 
-const supportFee = computed(() =>
-  isRemark.value ? totalNFTsPrice.value * SUPPORT_FEE_PERCENT : 0,
+const total = computed(
+  () => totalNFTsPrice.value + totalRoyalties.value + supportFee.value,
```

Comment:
are we sure they are all numbers?

Reply:
yes
![CleanShot 2023-11-23 at 18 07 42@2x](https://github.com/kodadot/nft-gallery/assets/44554284/db0eaaeb-b388-4618-8d9c-9a8118b9b036)

the only new value is support fee 

![CleanShot 2023-11-23 at 18 08 45@2x](https://github.com/kodadot/nft-gallery/assets/44554284/0f42a291-9971-42b2-bf8d-0aec27a0888a)

which is also a number